linked_list.py

```diff
@@ -0,0 +1,93 @@
+# _*_ encoding: utf-8 _*_
+"""Demonstrate linked list in python."""
+
+
+class LinkedList(object):
+    """Demonstrate linked list."""
+
+    def __init__(self, head=None):
+        """Initialize the list."""
+        self.head = head
+
+    def insert(self, val):
+        """Insert value at head of list."""
+        if type(val) == list:
```

Comment:
You shouldn't be passing in iterable. Per the instructions. This should take in a value, not a list.